dashboard/app/models/puzzle_rating.rb

```diff
@@ -28,7 +28,7 @@ class PuzzleRating < ActiveRecord::Base
   validates_uniqueness_of :user_id, :scope => [:script_id, :level_id], :allow_nil => true
 
   def PuzzleRating.enabled?
-    true
+    false
```

Comment:
Does this remove the puzzle rating from the UI or simply the database reads and writes? Ideally, from a UX perspective, I think we'd want to keep the UI but disable the connection to the database.